File: check.js
const fs = require('fs');
const lines = fs.readFileSync('C:/caas-dashboard/frontend/src/App.jsx', 'utf8').split('\r\n');
for (let i = 12980; i < 13010; i++) {
  if (lines[i] && lines[i].includes('content')) console.log((i+1)+':', lines[i].substring(0,100));
}

File: _check.py
with open(r"C:\caas-dashboard\frontend\src\ZertoPage.jsx","rb") as f:
    t = f.read().decode("utf-8-sig")
lines = t.split("\n")
print("Lines:", len(lines))
print("Last 5:")
for l in lines[-5:]: print(repr(l[:100]))
print("Has Drawer:", "OperationProgressDrawer" in t)
print("Has main return:", "return(<div style={{padding" in t)
print("Has export default:", "export default function" in t)